http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/KubernetesClusterComponentSpec.scala

```diff
@@ -92,7 +92,7 @@ class KubernetesClusterComponentSpec extends AnyFlatSpecLike with TestComponent
   it should "update async fields" in isolatedDbTest {
     val savedCluster1 = makeKubeCluster(1).save()
 
-    val newAsyncFields = KubernetesClusterAsyncFields(externalIp, networkFields)
+    val newAsyncFields = KubernetesClusterAsyncFields(loadBalancerIp, apiServerIp, networkFields)
```

Comment:
Yeah. The use case wasn't totally hammered down when we made the table. I figured it was worth specifying the type in case we need to support multiple and leo needs to decide how to route based on the AppType/Service type down the line. I'm not opposed to removing until it becomes relevant.